visma/io/parser.py

```diff
@@ -52,7 +52,20 @@ def tokensToLatex(eqTokens):
     """
     eqLatex = ""
     for token in eqTokens:
-        eqLatex += token.__str__()
+        if isinstance(token, Matrix):
+            eqLatex += "\\begin{bmatrix}"
```

Comment:
I think there will be a single backslash.
```suggestion
            eqLatex += "\begin{bmatrix}"
```